llm-example/llm.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import requests
+
+SYSTEM_PROMPT = os.getenv("SYSTEM_PROMPT") or ""
+
+
+def llm(prompt):
+    response = requests.post(
+        os.getenv("TRUEFOUNDRY_API_ENDPOINT"),
+        headers={
+            "Authorization": f"Bearer {os.getenv('TRUEFOUNDRY_API_KEY')}",
+        },
+        json={
+            "prompt": SYSTEM_PROMPT + prompt,
+            "model": os.getenv("TRUEFOUNDRY_MODEL"),
+            "temperature": 0.5,
+            "max_tokens": 200,
+            "top_p": 1.0,
+            "presence_penalty": 0.0,
+            "frequency_penalty": 0.0,
```

Comment:
You may skip all of these params. only pass max_tokens. Other default values will be used on its own.